app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -121,15 +122,17 @@
 
           ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
           ga('set', 'anonymizeIp', true);
+          <%= content_for(:before_pageview_js) %>
           <% if content_for?(:custom_pageview_fullpath) %>
             GOVUKAdmin.trackPageview('<%= content_for(:custom_pageview_fullpath) %>');
           <% else %>
             ga('send', 'pageview');
           <% end %>
         </script>
-      <% elsif Rails.env.development? %>
+      <% elsif allow_google_analytics_stub %>
         <script>
           if (console) {window.ga = function() {console.log.apply(console, arguments);};}
+          <%= content_for(:before_pageview_js) %>
```

Comment:
I am a bit unsure that there's not a better way to achieve this.  Not least because this is going to introduce console output in your tests when we do this.